lib/engine/company.rb

```diff
@@ -35,6 +35,8 @@ def min_bid
 
     def close!
       @closed = true
+
+      @abilities.each { |t, _| remove_ability(t) }
```

Comment:
this may break meat packing and steam boat because they get closed but their abilities don't go away will have to think

Reply:
Maybe abilities for companies could default to `"remove": "close"`, but meat packing and steam boat could have `"remove": "4"`, and this loop here could be

```
@abilities.each { |t, _| remove_ability(t) if t.remove == :close }
```